src/game.js

```diff
@@ -0,0 +1,115 @@
+const _ = require('lodash')
+const Betting = require('./betting')
+const currency = require('./currency')
+
+module.exports = class Game {
+  constructor() {
+    this.minBet = 10;
+    this.maxBet = 500;
+    this.CBet = 10;
+    this.minCBet = '👕'
+    this.midCBet = '👞'
+    this.maxCBet = '👖'
+    this.players = []
+    this.setup();
+    this.start();
+  }
+
+  report( theme, message ){
+    console.log(colors.green( theme+ '')+'> '+message);
+  }
+
+  prompt(ask){
+    return prompt.forString(ask)
+  }
+
+  promptForNumber(ask){
+    return prompt.forNumber(ask)
+  }
+
+  setup() {
+    clear()
+    if (!this.numberOfHumanPlayers){
```

Comment:
I am changing these conditions, as the team informed me that they only want one player and one AI Dealer. You will see my updates in the next push